gspread/models.py

```diff
@@ -986,6 +986,65 @@ def delete_row(self, index):
         }
 
         return self.spreadsheet.batch_update(body)
+    def add_protect_ranges(self, start_row_index,end_row_index,start_column_index,end_column_index, 
+                           editors_emails=None, description=None,  request_user_edit=False):
+        """"Add protect ranges into the selected worksheet. Only the editors can edit the protected ranges
+        :param start_row_index: Index of the start row.
+        :type start_row_index: int
+        :param end_row_index: Index of the end row.
+        :type end_row_index: int
+        :param start_column_index: Index of the start column.
+        :type start_column_index: int
+        :param end_column_index: Index of the end column.
+        :type end_column_index: int
+        :param editors_emails: List for more editors email
+        :type editors_emails: list
+        :param description: description for the protected ranges
+        :type description: str
+        :param request_user_edit: True if the user who requested this protected range can edit the protected area.
+        :type request_user_edit: boolean
+        """
+        emailAddresses = []
+        if (editors_emails != None) & (isinstance(editors_emails,list)):
```

Comment:
You might be able to greatly simplify this code by making the default for `editors_emails=[]`. Then you could simply extend `emailAddresses` with `editors_emails`